test/integration_testcase.go

```diff
@@ -54,6 +55,25 @@ func NewTestCase(t *testing.T, name string) *IntegrationTestCase {
 	return n
 }
 
+// CompareError compares expected error to error recived.
+func (tc *IntegrationTestCase) CompareError(err error) {
+	wantExists, want := len(tc.ErrorExpected) > 0, tc.ErrorExpected
+	gotExists, got := err != nil, ""
+	if gotExists {
+		got = err.Error()
+	}
+
+	if wantExists && gotExists {
+		if want != got {
+			tc.t.Errorf("expected error %s, got error %s", want, got)
+		}
+	} else if !wantExists && gotExists {
+		tc.t.Errorf("%s error raised where none was expected", got)
```

Comment:
`Fatalf` should be used on line 71. If an unexpected error occurs, the rest of the test is invalid, and should just end.